src/main/java/org/parabot/Landing.java

```diff
@@ -121,7 +119,7 @@ private static void parseArgs(String... args) {
                     } else {
                         i--;
                         for (int j = 0; j < 6; j++) {
-                            mac[j] = Byte.parseByte(args[++i], 16); // parses a hex
+                            mac[j] = (byte) Integer.parseInt(args[++i], 16); // parses a hex
```

Comment:
This seems like a mac address can only contain numbers. Is that true?

Reply:
Negative